src/Sulu/Component/HttpCache/HttpCache.php

```diff
@@ -93,6 +93,6 @@ private function getReverseProxyTtl(Response $response)
             return $maxAge - $response->getAge();
         }
 
-        return null;
+        return;
```

Comment:
Symfony code base: 321 `return;`, 136 `return null`. I see it as exiting the function which is not expected to have a return value.